src/EditorFeatures/VisualBasicTest/DocumentationComments/CodeFixes/RemoveDocCommentNodeCodeFixProviderTests.vb

```diff
@@ -0,0 +1,740 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.CodeFixes
+Imports Microsoft.CodeAnalysis.DiagnosticComments.CodeFixes
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Diagnostics
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.DocumentationComments.CodeFixes
+    Public Class RemoveDocCommentNodeCodeFixProviderTests
+        Inherits AbstractVisualBasicDiagnosticProviderBasedUserDiagnosticTest
+
+        Friend Overrides Function CreateDiagnosticProviderAndFixer(workspace As Workspace) As Tuple(Of DiagnosticAnalyzer, CodeFixProvider)
+            Return Tuple.Create(Of DiagnosticAnalyzer, CodeFixProvider)(
+                Nothing,
+                New VisualBasicRemoveDocCommentNodeCodeFixProvider())
+        End Function
+        
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveDuplicateParamTag)>
+        Public Async Function RemovesDuplicateParamTag() As Task
+            Dim initial =
+<File>
+Class Program
+    ''' <summary>
+    ''' 
+    ''' </summary>
+    ''' <param name="value"></param>
+    ''' [|<param name="value"></param>|]
+    Sub Fizz(ByVal value As Integer)
+    End Sub
+End Class
+</File>
+            Dim expected =
+<File>
+Class Program
+    ''' <summary>
+    ''' 
+    ''' </summary>
+    ''' <param name="value"></param>
+    Sub Fizz(ByVal value As Integer)
+    End Sub
+End Class
+</File>
+            
+            
+
+            Dim options = TestOptions.Regular.WithDocumentationMode(DocumentationMode.Diagnose)
+            Await TestAsync(initial.ToString(), expected.ToString(), options)
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveDuplicateParamTag)>
+        Public Async Function RemovesDuplicateParamTag_OnlyParamTags() As Task
+            Dim initial =
+<File>Class Program
+    ''' <param name="value"></param>
+    ''' [|<param name="value"></param>|]
+    Sub Fizz(ByVal value As Integer)
+    End Sub
+End Class
+</File>
+
+            Dim expected =
+<File>Class Program
+    ''' <param name="value"></param>
+    Sub Fizz(ByVal value As Integer)
+    End Sub
+End Class
+</File>
+            
+            Dim options = TestOptions.Regular.WithDocumentationMode(DocumentationMode.Diagnose)
+            Await TestAsync(initial.ToString(), expected.ToString(), options)
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveDuplicateParamTag)>
+        Public Async Function RemovesDuplicateParamTag_TagBelowOffendingParamTag() As Task
+            Dim initial =
+<File>Class Program
```

Comment:
note, vb has multiline string literals now.  So our general preference is to use those over xml literals.